huaweicloud/services/apm/resource_huaweicloud_apm_aksk.go

```diff
@@ -0,0 +1,167 @@
+package apm
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/internal/entity"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/internal/httpclient_go"
+	"io/ioutil"
+	"time"
+)
+
+func ResourceApmAkSk() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: ResourceApmAkSkCreate,
+		ReadContext:   ResourceApmAkSkRead,
+		DeleteContext: ResourceApmAkSkDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(5 * time.Minute),
+			Update: schema.DefaultTimeout(5 * time.Minute),
+			Delete: schema.DefaultTimeout(5 * time.Minute),
+		},
+		Schema: map[string]*schema.Schema{
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"access_key": {
```

Comment:
it seems that `access_key` and `secret_key` are computed only

Reply:
fixed